k8s/scripts/inspect.sh

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env bash
+
+INSPECT_DUMP=$(pwd)/inspection-report
+
+SVC_ARGS_DIR=/var/snap/k8s/common/args
+K8SD_STATE_DIR=/var/snap/k8s/common/var/lib/k8sd/state
+K8SD_BIN=/snap/k8s/current/bin/k8sd
+SBOM_FILE=/snap/k8s/current/bom.json
```

Comment:
Let's use some prefixes here so that we help reading the code below. Also, do not define things that are used once (`$K8S_SNAP` and `$K8S_SNAP_COMMON` can just be used there).

```
K8S_SNAP=/snap/k8s/current
K8S_SNAP_COMMON=/var/snap/k8s/common

# then 
K8S_ARGS_DIR="${K8S_SNAP_COMMON}/args"
...
```